protocol/app/upgrades/v5.0.0/upgrade.go

```diff
@@ -30,10 +31,41 @@ func perpetualsUpgrade(
 	}
 }
 
+func blockRateLimitConfigUpdate(
+	ctx sdk.Context,
+	clobKeeper clobtypes.ClobKeeper,
+) {
+	// Based off of https://docs.dydx.exchange/trading/rate_limits
+	blockRateLimitConfig := clobtypes.BlockRateLimitConfiguration{
```

Comment:
We should copy the old params from the existing state rather than setting them to constants.

We should initialize the new params to defaults rather than setting them to constants.